src/bundler.ts

```diff
@@ -30,32 +30,35 @@ import {FSUrlLoader} from 'polymer-analyzer/lib/url-loader/fs-url-loader';
 import constants from './constants';
 import * as matchers from './matchers';
 import PathResolver from './pathresolver';
-import ASTUtils from './ast-utils';
-
-
-function buildLoader(config: any) {
-  const abspath: string = config.abspath;
-  const excludes = config.excludes;
-  const fsResolver = config.fsResolver;
-  const redirects = config.redirects;
-  let root = abspath && path.resolve(abspath) || process.cwd();
-  let loader = new FSUrlLoader(root);
-  // TODO(garlicnation): Add noopResolver for external urls.
-  // TODO(garlicnation): Add redirectResolver for fakeprotocol:// urls
-  // TODO(garlicnation): Add noopResolver for excluded urls.
-  return loader;
+import * as ASTUtils from './ast-utils';
```

Comment:
style: name the library lowercase so it doesn't look like static methods at call sites.